plugins/services/services_inspector.rb

```diff
@@ -51,24 +51,55 @@ def summary
   private
 
   def inspect_systemd_services
-    output = @system.run_command(
+    services = list_unit_files.map do |line|
+      name, state = line.split(/\s+/)
+
+      Service.new(name: name, state: state)
+    end
+
+    list_unit_files(true).map { |line| line.split(/\s+/) }.each do |name, state|
+      services << Service.new(name: name, state: state) if state == "disabled"
```

Comment:
It's not very clear from the code why `disabled` services need to be treated separately here. Do you think we can clear this up somehow?